packages/sdk/connect/src/utils/is-cel2.ts

```diff
@@ -0,0 +1,13 @@
+import Web3 from 'web3'
+
+export const PROXY_ADMIN_ADDRESS = '0x4200000000000000000000000000000000000018'
```

Comment:
Where is this address from? Might be useful to leave a small note in the PR description, or a code comment about where the address is from.

I imagine Volpe or someone else told us, but might be handy to leave a note for posterity.

Reply:
Comment in https://github.com/celo-org/developer-tooling/pull/235/commits/98289d786caa8d372b2bc51c44372be2655991f9